util/election/election.go

```diff
@@ -47,12 +47,12 @@ var ErrNoMaster = errors.New("no master")
 
 // Factory encapsulates the creation of a MasterElection instance for a treeID.
```

Comment:
Please don't forget to update this comment. Mention generic `resourceID` here and `treeID` as an example.

Reply:
done